README.md

```diff
@@ -59,6 +59,24 @@ $ npm install
 $ ng serve
 ```
 
+## Docker Setup
+
+
+1. Build:
+  `docker build -t paradox-server`
+2. Deploy:
+
+Replace the paths on the left of the mappings to reflect your system:
+```docker
+  docker run -it --rm \
+    --name paradox-server \
+    -v /path/to/client:/luclient \
+    -v /path/to/client:/usr/src/paradox-server/client \
```

Comment:
Why does it need `/path/to/client` twice? Right now it's only using locale.xml and cdclient.fdb anyways

Reply:
I think this is due to me miss-understanding what was happening, will fix